clang/lib/Frontend/CompilerInvocation.cpp

```diff
@@ -3567,6 +3567,14 @@ void CompilerInvocationBase::GenerateLangArgs(const LangOptions &Opts,
       GenerateArg(Consumer, OPT_ftrigraphs);
   }
 
+  if (T.isOSzOS()) {
+    if (!Opts.ZOSExt)
+      GenerateArg(Consumer, OPT_fno_zos_extensions);
```

Comment:
I assume the default is -fno-zos-extensions, can you check why we need to explicitly pass this option?